person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './decorateor'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] As mentioned in the book class, you may need to add your `:rentals` to your `attr_accessor` to allow your person to have a rental. So please work on this issue.